core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/stringfunction/LpadExpression.kt

```diff
@@ -12,7 +13,15 @@ class LpadExpression(
     private val size: TypeExpression<NumberType>,
     private val prefix: TypeExpression<StringType>? = null,
 ) : TypeExpression<StringType>, FunctionOperator {
-    override fun toQueryString(): String = toFunctionQueryString(symbol = "LPAD", inStr, size, extra = prefix)
+    override fun toQuery(): DopeQuery {
+        val inStrDopeQuery = inStr.toQuery()
+        val sizeDopeQuery = size.toQuery()
+        val prefixDopeQuery = prefix?.toQuery()
+        return DopeQuery(
+            queryString = toFunctionQueryString(symbol = "LPAD", inStrDopeQuery, sizeDopeQuery, extra = prefixDopeQuery),
+            parameters = inStrDopeQuery.parameters + sizeDopeQuery.parameters + (prefixDopeQuery?.parameters ?: emptyMap()),
```

Comment:
there is an extension-function, `orEmpty()`: `...prefixDopeQuery?.parameters.orEmpty()`

Reply:
done